libs/@guardian/libs/src/logger/logger.test.ts

```diff
@@ -109,7 +109,7 @@ describe('Add and remove teams', () => {
 });
 
 describe('Team-based logging', () => {
-	const teams: TeamName[] = ['cmp', 'commercial', 'dotcom'];
+	const teams: Subscription[] = ['cmp', 'commercial', 'dotcom'];
```

Comment:
```suggestion
	const teams = ['cmp', 'commercial', 'dotcom'] satisfies Subscription[];
```

Reply:
what's the benefit of this?

Reply:
It narrows the type to only these specific three subscriptions, which actually brings little value in a test.